libraries/chain/controller.cpp

```diff
@@ -5257,6 +5259,7 @@ void controller::assemble_and_complete_block( block_report& br, const signer_cal
       valid_block_signing_authority);
 
    br = my->pending->_block_report;
+   br.start_time = start_time;
```

Comment:
This does not look correct. You want the total time for producing the block. This should be from the time of start_block() which should be what is in pending.

Reply:
Do you mean we don't need to do anything here, just use the `start_time` in pending as it is set to `now` at construction?

Reply:
I would assume so, yes.

Reply:
After the change, producing time looks close to 500 ms like
`info  2024-10-01T19:07:27.667 nodeos    controller.cpp:3461           commit_block         ] Produced block b29e040564b322f2... #100 @ 2024-10-01T19:07:28.000 signed by eosio [trxs: 0, lib: 98, net: 0, cpu: 100 us, elapsed: 291 us, producing time: 451393 us]`